src/admin/reducers/UserReducer.ts

```diff
@@ -23,12 +23,34 @@ import * as  UserActionsCreator  from 'admin/actions/UserActionsCreator';
 import AdminConstants from 'admin/constants/AdminConstants';
 
 const initialState = Immutable.from<User>({
-	name: '',
-	email: '',
-	role: UserRole.Journalist,
+	name: localStorage.getItem('rcUsername') || '',
+	email: localStorage.getItem('rcUserEmail') || '',
+	role: getRoleFromEnum(),
 });
 
+function getRoleFromEnum() {
+	let role: UserRole;
+	switch (localStorage.getItem('rcUserRole')) {
```

Comment:
pls refactor this. 
This 
case UserRole.Editor:
role = UserRole.Editor:
is not good

Reply:
ex.
let role: UserRole;
let storedRole =  localStorage.getItem('rcUserRole');
if (storedRole in UserRole) {
 role = storedRole;
} esle {
role = UserRole.Journalist;
}

Reply:
I would be very pleased, but TS doesn't allow me to do it

Reply:
Maybe we can do it in another way. switch..case looks not good